shell.nix

```diff
@@ -25,7 +35,8 @@ ghc       = ghcOrig.override (oldArgs: {
 ###
 drvf =
```

Comment:
We could, but do we need to do it now?  There are many improvements we can make..